library/core/src/slice/mod.rs

```diff
@@ -3668,7 +3668,8 @@ impl<T> [T] {
     /// assert_eq!(a, ['a', 'c', 'd', 'e', 'b', 'f']);
     /// ```
     #[stable(feature = "slice_rotate", since = "1.26.0")]
-    pub fn rotate_left(&mut self, mid: usize) {
+    #[rustc_const_unstable(feature = "const_slice_rotate", issue = "none")]
+    pub const fn rotate_left(&mut self, mid: usize) {
```

Comment:
Please create a tracking issue

Reply:
Created an issue and added it to the PR description.